Assets/Scripts/StandardSamples/UI/MatchMaking/UISessionsMatchmakingMenu.cs

```diff
@@ -403,5 +434,112 @@ public void OnSessionRefresh(Session session, SessionDetails details)
 
             uiEntry.SetUIElementsFromSessionAndDetails(session, details, this);
         }
+
+        #region UIFriendInteractionSource Implementations
+
+        /* 
+         * REGION NOTES:
+         * 
+         * The UIFriendsMenu popout uses the base class UIFriendInteractionSource to facilitate interactions.
+         * 
+         * NOTE: This current implementation is not complete.
+         * There is functionality suggested by the existance of this window that need to be added.
```

Comment:
*needs

Reply:
Yup, that's a valid grammar fixup. Changed.